wave-2-game.rb

```diff
@@ -0,0 +1,124 @@
+require_relative 'lib/scrabble'
+
+module Scrabble
+  class Game
+    def initialize
+      @words = []
+      @players = setup_players
+
+    end
+
+    def play
+      start
+
+      while continue?
+        @players.each do |player|
+          puts "It is #{player.name}'s turn"
+          player_word = get_word_for player
+          player_has_won = !player_word
+          
+          if player_word
+            print_score player_word
+          end
+
+          if player_has_won
+            break
+          end
+        end
+      end
+
+      conclude
+    end
+
+    private
+
+    def setup_players
+      puts "How many players do you have?"
+      num_of_players = get_int_input
+
+      players = []
+      num_of_players.times do |player|
+        players << Player.new(get_player_name player)
+      end
+      return players
+    end
+
+    def start
+      puts "Welcome to our Scrabble game!"
+    end
+
+    def continue?
+      return true if @words.length == 0 # haven't started playing yet
+      @players.each do |player|
+        if player.won?
+          crown_winner player
+          return false
+        end
+      end
+
+      puts "Would you like to play another round? (Y/N)"
+      continue = gets.chomp
+      (continue == "Y") ? true : false
+    end
+
+    def get_word_for player
+      puts "Enter a word to score:"
+      word = gets.chomp
+      @words << word
+
+      keep_playing = player.play word
```

Comment:
This is good - I wonder if we should encourage the Player's `won?` method to be a private method to only be used within the Player class?